src/app/static/src/app/components/modelOptions/ModelOptions.vue

```diff
@@ -12,6 +12,13 @@
                       :required-text="requiredText"
                       :select-text="selectText"></dynamic-form>
 
+        <div v-if="hasOptionsError" id="model-option">
+            <div v-if="!otherError" class="error-message" id="model-option-error">
```

Comment:
As per chat with @r-ash we can take out this markup dealing with special case to show key, and all associated code (e.g. otherError, locale strings), and just retain the `<error-alert>`, and trust hintr to pass us a sensible error detail.